cellmlmanip/units.py

```diff
@@ -267,9 +267,6 @@ def get_conversion_factor(self, from_unit, to_unit):
         cf = self.convert(1 * from_unit, to_unit).magnitude
         if isinstance(cf, numbers.Number) and math.isclose(cf, 1.0):
             return 1
-        elif isinstance(cf, sympy.Mul) and 1.0 in cf.args:
```

Comment:
This code wasn't being triggered by the tests, seems not to be necessary anymore.